performance-measure/src/test/java/org/sonarsource/performance/measure/DurationMeasureMergerTest.java

```diff
@@ -0,0 +1,226 @@
+    Path tmpLatestAnalysisFolder = tmpDirectory.resolve("events").resolve("6.15.0.25780");
+    Files.createDirectories(tmpLatestAnalysisFolder);
+
+    Path previousReleaseFolder = Paths.get("src", "test", "resources", "events", "6.15.0.25600");
+    Path latestAnalysisFolder = Paths.get("src", "test", "resources", "events", "6.15.0.25780");
+    merger.withMaxScoreThreshold(1.06)
+      .compareWithRelease(previousReleaseFolder, latestAnalysisFolder, tmpLatestAnalysisFolder, tmpGlobalPerformanceScoreFile);
+
+    assertThat(tmpGlobalPerformanceScoreFile).hasContent("" +
+      "{\n" +
+      "  \"scoreOverstepThreshold\": false,\n" +
+      "  \"score\": \"105.1%\",\n" +
+      "  \"link\": \"https://github.com/SonarSource/peachee-languages-statistics/blob/sonar-java/events/6.15.0.25780/performance.score.json\"\n" +
+      "}");
+  }
+
+  @Test
+  void compare_with_fake_previous_release(@TempDir Path tmpDirectory) throws IOException {
+    Path tmpGlobalPerformanceScoreFile = tmpDirectory.resolve("performance.score.json");
+    Path tmpLatestAnalysisFolder = tmpDirectory.resolve("events").resolve("6.15.0.25600");
+    Files.createDirectories(tmpLatestAnalysisFolder);
+
+    Path previousReleaseFolder = Paths.get("src", "test", "resources", "events", "6.14.0.25463");
+    Path latestAnalysisFolder = Paths.get("src", "test", "resources", "events", "6.15.0.25600");
+    merger.compareWithRelease(previousReleaseFolder, latestAnalysisFolder, tmpLatestAnalysisFolder, tmpGlobalPerformanceScoreFile);
+
+    Path expectedScoreFile = latestAnalysisFolder.resolve("performance.score.json");
+    Path actualScoreFile = tmpLatestAnalysisFolder.resolve("performance.score.json");
+    assertSameTextContent(expectedScoreFile, actualScoreFile);
+
+    assertThat(tmpGlobalPerformanceScoreFile).hasContent("" +
+      "{\n" +
+      "  \"scoreOverstepThreshold\": false,\n" +
+      "  \"score\": \"93.5%\",\n" +
+      "  \"link\": \"https://github.com/SonarSource/peachee-languages-statistics/blob/sonar-java/events/6.15.0.25600/performance.score.json\"\n" +
+      "}");
+
+    assertThat(logger.logs().replaceAll("\\R", "\n")).isEqualTo("" +
+      "[INFO] Compare Performances between release 6.14.0.25463 and latest 6.15.0.25600\n" +
+      "[INFO] Writing: " + actualScoreFile + "\n" +
+      "[INFO] Writing: " + tmpGlobalPerformanceScoreFile + "\n");
+  }
+
+  @Test
+  void can_not_find_any_performance_file(@TempDir Path destDirectory) throws IOException {
+    Path latestAnalysisFolder = Paths.get("src", "test", "resources", "events", "6.15.0.25780");
+    merger
+      .forPerformanceFileName("invalid.performance.measure.json")
+      .mergeProjectPerformances(latestAnalysisFolder, destDirectory);
+    assertThat(logger.logs().replaceAll("\\R", "\n")).isEqualTo("" +
+      "[INFO] Merge Project Performances of 6.15.0.25780\n" +
+      "[WARNING] Can't find any 'invalid.performance.measure.json' in src/test/resources/events/6.15.0.25780\n");
```

Comment:
My Windows machine doesn't like this assertion as path separators are `\` instead of `/` there 😢